### PR TITLE
ID21498: Added lviv config

### PR DESCRIPTION
**Pull request by sviridovartem**

### CODE REVIEW

File: docker-compose-lviv.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  webviewer:
+    image: node:14.5.0
+    volumes:
+      - ./webviewer:/client
+    working_dir: /client
+    expose:
+      - 3000
+    ports:
+      - "3000:3000"
+    command: bash -c "npm install && npm start"
```
Comment (adolgarev) [inline docker-compose-lviv.yml:42]: as we made a custom docker-compose-lviv.yml lets get rid of node dev server, see how it is done in https://github.com/QuantuMobileSoftware/geoap/blob/master/docker-compose-prod.yml.

**Review by adolgarev — COMMENTED** [review]
The basis for docker-compose-lviv.yml should be docker-compose-prod.yml. The difference in domain names and SMTP backend

File: webapplication/tasks.py [hunk 2]
```diff
@@ -88,3 +88,16 @@ def test(ctx):
...
+    init_db(ctx, recreate_db=False)
+    collect_static_element(ctx)
+
+    thread_cron = threading.Thread(target=devcron, args=(ctx,))
+    thread_cron.start()
+
+    thread_nb_executor = threading.Thread(target=run_notebook_executor, args=(ctx,))
+    thread_nb_executor.start()
+
+    ctx.run('uwsgi --ini uwsgi.ini')
```
Comment (adolgarev) [inline webapplication/tasks.py:103]: what is the difference with run_prod?

Comment (sviridovartem) [reply webapplication/tasks.py:103]: fixed

File: docker-compose-lviv.yml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+      - "5433:5432"
+    environment:
+      - POSTGRES_DB=sip_db
+      - POSTGRES_USER=sip
+      - POSTGRES_PASSWORD=eiJ2koo6
+
+  webapplication:
+    build:
+      context: ./webapplication
+      dockerfile: Dockerfile
```
[the remark this answers is not in the document]
Comment (sviridovartem) [reply docker-compose-lviv.yml:19]: done

**Review by adolgarev — APPROVED** [review]

**Pull request merged**